jetty-server/src/main/java/org/eclipse/jetty/server/handler/ResourceHandler.java

```diff
@@ -143,7 +143,7 @@ public MimeTypes getMimeTypes()
     public Resource getResource(String path)
     {
         if (LOG.isDebugEnabled())
-            LOG.debug("{} getResource({})", _context == null ? _baseResource : _context, _baseResource, path);
+            LOG.debug("{} getResource({})", _context == null ? _baseResource : _context, path);
```

Comment:
This one had base vs context resource, then resourcebase, then path.  Not sure what the correct step here should have been.
Almost thinking it should just be ...

``` java
LOG.debug("getResource({}) baseResource={}, context={}", path, _baseResource, _context);
```

WDYT?

Reply:
I'm thinking it looks like we just want either the context, if there is one, or the baseResource otherwise, plus the path arg to the method, so the change in your commit is correct.